lib/osf-components/addon/components/search-result-card/template.hbs

```diff
@@ -97,11 +95,11 @@
                 </div>
             {{/if}}
         </div>
-        <CpPanel local-class='cp-panel' id={{secondaryMetadataPanelId}} @open={{this.isOpenSecondaryMetadata}} as |panel|>
-            <panel.body>
-                <hr>
-                {{component this.secondaryMetadataComponent result=@result}}
-            </panel.body>
-        </CpPanel>
+        {{component this.secondaryMetadataComponent
+            wrapperClass=(local-class 'cp-panel')
+            wrapperId=secondaryMetadataPanelId
+            result=@result
+            isOpen=this.isOpenSecondaryMetadata
+        }}
```

Comment:
i gather that `<CpPanel>` was moved inside each `secondaryMetadatComponent` so that `user-secondary-metadata` knows when it's toggled and can fetch the user, but now there's duplicate `<CpPanel>` invocations and a more complicated shared interface -- instead, could `user-secondary-metadata` start fetching the user from its `constructor`? (or is it bad practice to start tasks from `constructor`? been a while)

Reply:
Yea, this refactor took a little bit of doing. I had to move the `<CpPanel>`into the secondaryMetadata components, since the component would be destroyed every time the card was expaneded/closed if the `<CpPanel>` lived on the `search-result-card` component. If we tied the user fetching to the `constructor`, it would result in fetching the user every time the card is expanded. Also ran into trouble using splattributes on the secondaryMetadataComponents when invoking it using the `{{component}}` helper which was interesting